packages/flutter/lib/src/rendering/debug_overflow_indicator.dart

```diff
@@ -253,6 +253,10 @@ mixin DebugOverflowIndicatorMixin on RenderObject {
           // TODO(jacobr): this line is ascii art that it would be nice to
           // handle a little more generically in GUI debugging clients in the
           // future.
+          if (debugCreator?.element != null) {
```

Comment:
Do we have asserts in place to make sure that debugCreator gets set in debug mode?

I realize this will still be needed, unless we wrapped the whole new block in an asset (and probably used a `List` instead of `sync*`.

Reply:
should we assert when a render object is created or assert before we report the error?

Reply:
I think earlier is better.  I'm a bit curous about how much that would break though.

Reply:
I will see if i can do a check during renderobject creation, hopefully it is not a breaking change.

Reply:
Is there some kind of test we could write to assert that this gets set in debug mode for everything in the framework?